### DISCUSSION TITLE
Pre-release branch to production

### DISCUSSION THREAD

**Discussion by Jaanilj** [semantic-release/semantic-release#2393] [Release workflow] [answered]
Hi!

Sorry for quite a rudimentary question, but I haven't found a good answer to this from any resource.

My problem is quite simple 😄 

- I have `master` and `beta` branches in my remote and `.releaserc.json` configs defined.
- I started to work on a feature that I merged to `beta` branch for prerelease integration testing.
  - The merge triggered a beta release and a commit to update to `package.json` and `CHANGELOG.md` 👍  Sweet! Everything was fine and dandy.

Now I want to merge the `beta` prerelease branch to `master`. I'm using Gitlab and creating an MR from `beta` to `master` which pulled commits from `beta` and the top commit in the MR is the latest `beta` commit for the release update with message `"chore(release): 1.2.0-beta.1 [skip ci]"`.

This means that the MR pipeline doesn't get triggered, because it's marked a `[skip ci]` so I cannot, this way, merge `beta` to `master`. How do ppl handle this?

From [the docs](https://github.com/semantic-release/semantic-release/blob/master/docs/recipes/release-workflow/pre-releases.md#publishing-the-200-beta-release-to-the-default-distribution-channel) I see that:

> To do so we need to merge our changes made on beta into master. As beta and master branches have diverged, this merge might require resolving conflicts. Once the conflicts are resolved and the merge commit is pushed to master, semantic-release will release version 2.0.0 on the dist-tag @latest.

I could create a merge commit to a new branch and merge that to `beta`, but that seems more laborious than required.
Should this `chore(release): ... [skip ci]` commit in `beta` even be merged to the `master`? It is a bit sus as it would change the `package.json` version as well, temporarily, before a new release is cut in the `master` branch with its own `chore(release): ... [skip ci]` commit.

**Comment by msclock** [comment, marked as answer]
Hi! There is a workaround. Open this `Skipped pipelines are considered successful` can continue the pr when using GitLab.
![image](https://user-images.githubusercontent.com/43399351/190570802-56b22434-f6bb-4d43-98d0-95d86df4af5b.png)
